src/tree/CosmosDBAccountProvider.ts

```diff
@@ -28,7 +27,7 @@ export class CosmosDBAccountProvider implements IChildProvider {
     }
 
     public async loadMoreChildren(node: IAzureNode): Promise<IAzureTreeItem[]> {
-        const client = getCosmosDBManagementClient(node.credentials, node.subscriptionId);
+        const client = new CosmosDBManagementClient(node.credentials, node.subscriptionId, node.environment.resourceManagerEndpointUrl);
```

Comment:
Hi, here are a couple of things I have noticed: 
1. The change effectively removes the userAgent. The userAgent helps _Azure_ collect the telemetry, not us. The intent of this is not very clear.  
2. This change affects _all clients created_, not just sovereign accounts. 

I would suggest  the following: 
```
export function getCosmosDBManagementClient(credentials: ServiceClientCredentials, subscriptionId: string, baseUri: string, isSovereign: boolean): CosmosDBManagementClient {  
//difference in function definition - added baseUri and a boolean flag. 
}
----

public loadMoreChildren(....){
//logic to check if it is a sovereign account
getCosmosDBManagementClient(node.credentials, node.subscriptionId, , node.environment.resourceManagerEndpointUrl, account.isSovereign);
....
}
```

Reply:
@yujhongmicrosoft It looks like from https://github.com/Microsoft/vscode-azureappservice/pull/554/files that you simply intended to add an extra argument to the new CosmosDBManagementClient call. Please add this argument to getCosmosDBManagementClient instead of replacing it with a call to `new CosmosDBManagementClient ()`